Package.swift

```diff
@@ -20,14 +20,7 @@ let package = Package(
         .target(
             name: "Valet",
             dependencies: []),
-        .target(
-            name: "LegacyValet",
```

Comment:
This target wasn't used by anything in the Package, so it seems reasonable to remove it.